src/hf_hydrodata/point.py

```diff
@@ -646,6 +661,8 @@ def get_site_variables(*args, **kwargs):
         elif options["dataset"] == "scan":
             dataset_query = """ AND site_type == ?"""
             param_list.append("SCAN station")
+        elif options["dataset"] == "fan_2013":
+            dataset_query = """ AND fan_2013 == 1 AND var_id == 26"""
```

Comment:
Nit: Couldn't this just be:
```suggestion
            dataset_query = " AND fan_2013 == 1 AND var_id == 26"
```

Reply:
You're right and thanks for pointing this out! I think my default when writing SQL in Python code is to use the `"""` quotes but that is only needed if it spans multiple lines. I will update these back to just a single set.